lib/streamlit/runtime/caching/hashing.py

```diff
@@ -451,6 +451,46 @@ def _to_bytes(self, obj: Any) -> bytes:
                 # it contains unhashable objects.
                 return b"%s" % pickle.dumps(obj, pickle.HIGHEST_PROTOCOL)
 
+        elif type_util.is_type(obj, "polars.series.series.Series"):
+            import polars as pl  # type: ignore[import-not-found]
+
+            obj = cast(pl.Series, obj)
+            self.update(h, obj.estimated_size())
+            self.update(h, obj.dtype.__hash__())
```

Comment:
Is `obj.dtype.__hash__` stable across reruns / new sessions? And isn't this the same as `hash(obj.dtype)`

Reply:
Good catch, no it isn't 

rewrote it with stable way